redpanda/redpanda.go

```diff
@@ -272,30 +272,51 @@ func (r *Redpanda) Configure(ctx context.Context, request provider.ConfigureRequ
 	// then Redpanda will correctly pick up the variables as well.
 	azureSubscriptionID := firstNonEmptyString(
 		conf.AzureSubscriptionID.ValueString(),
-		os.Getenv("ARM_SUBSCRIPTION_ID"))
+		os.Getenv("ARM_SUBSCRIPTION_ID"),
+		os.Getenv("AZURE_SUBSCRIPTION_ID"))
 	gcpProjectID := firstNonEmptyString(
 		conf.GcpProjectID.ValueString(),
 		os.Getenv("GOOGLE_PROJECT"),
 		os.Getenv("GOOGLE_CLOUD_PROJECT"),
 		os.Getenv("GCLOUD_PROJECT"),
 		os.Getenv("CLOUDSDK_CORE_PROJECT"))
+	azureClientID := firstNonEmptyString(
+		os.Getenv("AZURE_CLIENT_ID"),
+		os.Getenv("ARM_CLIENT_ID"))
+	azureClientSecret := firstNonEmptyString(
+		os.Getenv("AZURE_CLIENT_SECRET"),
+		os.Getenv("ARM_CLIENT_SECRET"))
+	azureTenantID := firstNonEmptyString(
+		os.Getenv("AZURE_TENANT_ID"),
+		os.Getenv("ARM_TENANT_ID"))
+	googleCredentials := os.Getenv("GOOGLE_CREDENTIALS")
+	googleCredentialsBase64 := os.Getenv("GOOGLE_CREDENTIALS_BASE64")
```

Comment:
Not a blocker: I always wonder if there is base64 why should we use the other one? Maybe you know.

Reply:
IMO we shouldn't. I support the standard because I want to avoid issues for end users but IMO the base64 should absolutely be the default.